server/claim-review/claim-review.service.ts

```diff
@@ -86,6 +86,7 @@ export class ClaimReviewService {
             { $limit: parseInt(pageSize) }
         );
 
+        // error when uses aggregation
```

Comment:
question: the CI didn't file, should it?

Reply:
Maybe I didn't expressed my self well, it doesn't return an error. The problem is that the aggregation return empty  when the test is ran.